source/usage-examples/insertMany.txt

```diff
@@ -30,8 +30,8 @@ Click `here <{+example+}/insertMany.go>`__ to see a fully runnable example.
 Expected Result
 ---------------
 
-After you run the code example, two new documents will be present in the
-collection with an ``_id`` field that matches the previous output:
+After you run the preceding example, you can find the inserted documents
```

Comment:
Nitpick: 

Seems like "following documents" is used in other places, but not here.

```suggestion
After you run the preceding example, you can find the following inserted documents
```

Applies to a few instances.